writer/licence_writer_test.go

```diff
@@ -15,8 +17,8 @@ func TestCheckIfLicenceExists_LicenceExists(t *testing.T) {
 	}
 }
 
-func TestCheckIfLicenceExists_LicenceNotFound(t *testing.T) {
-	ok, err := CheckIfLicenceExists("test1.java")
+func TestCheckIfFileContainsLicenceAlready_LicenceNotFound(t *testing.T) {
```

Comment:
It would be better to have those files inside our original project structure. Because we have tests but we have not files so that means tests will be failed when we run.

Reply:
This is a development enviremont so I think it is not good to keep test file in project structure. Because final product will not contains these test files.

Reply:
Then embed file contents into tests as strings. Create temporary files when test runs.